ossdbs/fem/mesh.py

```diff
@@ -31,8 +34,14 @@ def __init__(self, geometry: netgen.occ.OCCGeometry, order: int) -> None:
 
     def generate_mesh(self, meshing_parameters: dict) -> None:
         """Generate NGSolve mesh."""
-        netgen_mp = self._meshing_parameters(meshing_parameters)
+        netgen_mp = self._meshing_parameters(meshing_parameters["MeshingHypothesis"])
```

Comment:
I'm not a fan of this change, what's the reason behind it?

Reply:
As we now provide the mesh_settings to the method, we need to specify the Meshinghypothesis as the parameter for the `_meshing_parameters`.